jenkins/ci.opensuse.org/openstack-prepare-staging.yaml

```diff
@@ -0,0 +1,148 @@
+- job:
+    name: 'openstack-prepare-staging'
+    node: cloud-trackupstream
+
+    parameters:
+      - string:
+          name: openstack_project
+          default: None
+          description: name of the main project that should be updated with packages from its staging project
+
+    builders:
+      - shell: |
+          #!/usr/bin/python2
+          import osc.commandline
+          import osc.core
+          import sys
+          import os
+
+          try:
+              from xml.etree import cElementTree as ET
+          except ImportError:
+              import cElementTree as ET
+
+          # copied from newer osc/core.py
```

Comment:
newer than what? and why copied?

Reply:
Copied from https://github.com/openSUSE/osc/pull/465 which is newer than the installed version of osc. See the TODO comments below.